packages/terra-table/src/utils/actionsUtils.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * The method that checks if at least one of the columns has actions object with two parameters - text label and onCall method.
+ * @param {PropTypes.arrayOf(columnShape)} columns - array of data grid columns
+ * @returns bool
+ */
+const checkForColumnActions = (columns) => (!!columns.find(column => column.action));
```

Comment:
Can we rename this to `hasColumnActions` to be more in line with how Terra generally names booleans/functions returning booleans?

Reply:
Renamed in https://github.com/cerner/terra-framework/pull/2024/commits/d050bcd59ec5d4c9111847c976b1d45373400a05